docs/source/platform/operation-registry.md

```diff
@@ -290,6 +290,50 @@ const server = new ApolloServer({
 });
 ```
 
+## Migrating from 0.1
+
+#### On servers that safelist operations
+
+- Upgrade `apollo-server` to `2.6.2` and `apollo-server-plugin-operation-registry` to `0.2.0-alpha.1`
+
+#### To target graph variants other than "current"
```

Comment:
I think it would be helpful to add that "current" is the default, no tag. Maybe something like:

```
#### To target graph variants other than "current"(default, no tag)
```